### DISCUSSION TITLE
Account recovery administration not forcing master password update

### DISCUSSION THREAD

**Discussion by WolkanProv** [dani-garcia/vaultwarden#4552] [Q&A]
According to the docs: https://bitwarden.com/help/account-recovery/#after-a-recovery 

I should be getting this prompt, forcing me to change my password after a recovery. However, I never receive this prompt. Is this currently not implemented, or am I overlooking a setting somewhere?

**Comment by stefan0xC** [comment]
That part has not been implemented.
https://github.com/dani-garcia/vaultwarden/blob/0fe93edea6cb8d4b30416a6d319164f8828ad8b7/src/api/identity.rs#L299
https://github.com/dani-garcia/vaultwarden/blob/0fe93edea6cb8d4b30416a6d319164f8828ad8b7/src/db/models/user.rs#L259

Possible related issue #4507

**Comment by ekadin-mtc** [comment]
Just came looking for info on this.

Are there any plans to implement this?

**Comment by zain19** [comment]
as of release 136.0 master passwords recovered by organization admin not being re-prompted to change their password after the recovery this applies to (webvault , browser extensions , desktop app), also **"Require existing members to change their passwords"** flag in the master password policy does not prompt the non-compliant users to change their passwords upon new login rather the user is prompted to change the password when unlocking the vault this applies to (browser extensions , desktop app) , webvault never prompt the user to change password, am i right? and if so , what are the plans for implementing this?
All thanks to your great efforts.
